src/config/texts.js

```diff
@@ -6,13 +6,21 @@ export const texts = {
       active: 'Ausgewählt',
       inactive: 'Nicht ausgewählt'
     },
+    dropDownMenu: {
+      closed: 'geschlossen',
+      open: 'geöffnet'
+    },
     searchInputIcons: {
       delete: 'Suche löschen',
       search: 'Suche'
     },
     secureInputIcons: {
       invisible: 'Kennwort verstecken',
       visible: 'Kennwort sichtbar machen'
+    },
+    tabs: {
+      active: 'Aktiviert',
+      inactive: 'Nicht aktiviert'
```

Comment:
i suggest `aktiviert` and `nicht aktiviert` as these are not nouns in German and to be inline with `geschlossen` and `geöffnet`

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/f235b9769c1abcef45eb059543897a509bea090f